x-pack/test/send_search_to_background_integration/services/send_to_background.ts

```diff
@@ -47,6 +48,16 @@ export function SendToBackgroundProvider({ getService }: FtrProviderContext) {
       await testSubjects.click('backgroundSessionIndicatorViewBackgroundSessionsLink');
     }
 
+    public async deleteAllBackgroundSessions() {
```

Comment:
Hmm, is this problematic if other tests set up background sessions prior to this being called? Also, I'm wondering if this will fail when `.kibana_*` are treated as system indices. Could we use the routes for finding/deleting background sessions instead?

Reply:
Not having this would be problematic if other tests set up background sessions. If any test creates sessions without removing them, it doesn't let the "No items found" test to start from scratch. Any test that creates sessions should call a cleanup function afterward

> Could we use the routes for finding/deleting background sessions instead?

That sounds reasonable and will make more prod code used for the automated tests.

Reply:
https://github.com/elastic/kibana/pull/81707/commits/f632a4c37062f01ff35c9638e8ac475f25632806